app/tools/formTool.js

```diff
@@ -0,0 +1,37 @@
+function formBuilder(...args) {
```

Comment:
Good use of the ...args aspect of ES6 but why define the function according to ES5 standards? It would be better written as an arrow function to have unity with the newer standard.